sql_r_tutorial.Rmd

```diff
@@ -0,0 +1,192 @@
+---
+output:
+  html_document: default
+  pdf_document: default
+---
```

Comment:
Please remove these 5 lines so that the single hashtag # header is at the top of the file.